src/styles/main.scss

```diff
@@ -0,0 +1,63 @@
+@import './utils/variables';
+@import './utils/mixins';
+
+body {
+  margin: 0;
+  min-height: 100vh;
+
+  @include flex-center;
```

Comment:
According to the [Code Style Rules](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), the mixin should be included at the start of the block. Please, move the @include flex-center; to the top of the body selector.